src/services/services.ts

```diff
@@ -1834,7 +1834,7 @@ namespace ts {
             const fileContents = sourceFile.text;
             const result: TodoComment[] = [];
 
-            if (descriptors.length > 0) {
+            if (descriptors.length > 0 && !isNodeModulesFile(fileName)) {

```

Comment:
Is `fileName` already normalized by this point?  If not, is `sourceFile.fileName`?